### PR TITLE
Note location of include files

### LINKED ISSUE #37143: Missing header file locations for .NET Hosting in C++
I was following the [MSDN page on .NET Hosting](https://learn.microsoft.com/en-us/dotnet/core/tutorials/netcore-hosting), and they linked to this [sample C++ app](https://github.com/dotnet/samples/blob/main/core/hosting/src/NativeHost/nativehost.cpp) to demonstrate hosting a .NET (C#) program from a native app (C++).

I noticed these three includes:
```cpp
#include <nethost.h>
#include <coreclr_delegates.h>
#include <hostfxr.h>
```

I found them available at the following 3 links:
- https://github.com/dotnet/runtime/blob/main/src/native/corehost/nethost/nethost.h
- https://github.com/dotnet/runtime/blob/main/src/native/corehost/coreclr_delegates.h
- https://github.com/dotnet/runtime/blob/main/src/native/corehost/hostfxr.h

Perhaps this can be included in the docs page, cause otherwise, I wouldn't be able to do anything with .NET hosting!
Thanks for your help :) I'm looking forward to scripting C# over my C++ codebase for the first time in a cross-platform app.

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 83da7030-b5f1-72ad-71f6-424ff45b8b7f
* Version Independent ID: a98926cc-64ab-2911-e4b8-1eaf4573dfe4
* Content: [Write a custom .NET runtime host - .NET](https://learn.microsoft.com/en-us/dotnet/core/tutorials/netcore-hosting)
* Content Source: [docs/core/tutorials/netcore-hosting.md](https://github.com/dotnet/docs/blob/main/docs/core/tutorials/netcore-hosting.md)
* Product: **dotnet-fundamentals**
* GitHub Login: @mjrousos
* Microsoft Alias: **tdykstra**

### PR DESCRIPTION
**Pull request by tdykstra**
Fixes #37143 

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [docs/core/tutorials/netcore-hosting.md](https://github.com/dotnet/docs/blob/ad021a018c384fe774eb815a2fe9d07a6f16c888/docs/core/tutorials/netcore-hosting.md) | [Write a custom .NET host to control the .NET runtime from your native code](https://review.learn.microsoft.com/en-us/dotnet/core/tutorials/netcore-hosting?branch=pr-en-us-38294) |

### CODE REVIEW

File: docs/core/tutorials/netcore-hosting.md [hunk 1]
```diff
@@ -45,6 +45,20 @@ The `hostfxr` library is found using `get_hostfxr_path` API from `nethost` libra
...
+
+```cpp
+#include <nethost.h>
+#include <coreclr_delegates.h>
+#include <hostfxr.h>
+```
+
+These files can be found at the following locations:
+
+* `https://github.com/dotnet/runtime/blob/main/src/native/corehost/nethost/nethost.h`
```
Comment (gewarren) [inline docs/core/tutorials/netcore-hosting.md:58]: Any reason not to make these actual links?

**Review by gewarren — APPROVED** [review]

File: docs/core/tutorials/netcore-hosting.md (hunk 1 above)
Comment (tdykstra) [reply docs/core/tutorials/netcore-hosting.md:58]: Markdownlint complains about naked URLs, so I would have to repeat the URL or not show it, and neither of those options seemed appealing.

Comment (gewarren) [reply docs/core/tutorials/netcore-hosting.md:58]: You can do this:

```suggestion
* <https://github.com/dotnet/runtime/blob/main/src/native/corehost/nethost/nethost.h>
```

Comment (tdykstra) [reply docs/core/tutorials/netcore-hosting.md:58]: Oh, I forgot about that! I'll change it.

File: docs/core/tutorials/netcore-hosting.md [hunk 2]
```diff
@@ -45,6 +45,20 @@ The `hostfxr` library is found using `get_hostfxr_path` API from `nethost` libra
...
+* `https://github.com/dotnet/runtime/blob/main/src/native/corehost/coreclr_delegates.h`
+* `https://github.com/dotnet/runtime/blob/main/src/native/corehost/hostfxr.h`
```
Comment (tdykstra) [inline docs/core/tutorials/netcore-hosting.md:60]:
```suggestion
* <https://github.com/dotnet/runtime/blob/main/src/native/corehost/coreclr_delegates.h>
* <https://github.com/dotnet/runtime/blob/main/src/native/corehost/hostfxr.h>
```

**Comment by tdykstra** [comment]
Thanks for the review!

**Pull request merged**